fern/pages/fern-docs/content/customize-api-ref.mdx

```diff
@@ -0,0 +1,190 @@
+---
+title: Customize API Reference layout
+description: Customize your API Reference's naming, ordering, and structure.  
+---
+
+When you include an API (either the Fern or an OpenAPI definition) in your `docs.yml` file, you can customize how the endpoints and sub-packages are displayed in the sidebar navigation. By default, the reference will generate a navigation hierarchy based on the structure of the API spec, but several customizations can be configured.
+
+## Ordering the API Reference
+
+### Flattening sections
```

Comment:
This feels like you're describing a change from the default. My question while reading this section is "what's the default?" 

As a user, I want to see the before/after. To do that, I'd want to start with:

```
navigation:
  - api: API Reference
```

and then show the changes that are possible from there. What do you think about having a caption below each image that has a preview link where they can actually pull up each variation?

Reply:
The first page of the API Reference section within docs covers this (including an overview of the top-level configurations. I'll work on both pages so they synthesize well